frb_rust/src/ffi/web.rs

```diff
@@ -371,6 +370,14 @@ impl PortLike {
     }
 }
 
+impl Clone for PortLike {
+    fn clone(&self) -> Self {
+        PortLike {
+            obj: JsValue::clone(&self),
+        }
+    }
+}
```

Comment:
uber-nit: Hmm so it is not derivable anymore?

Reply:
There was a local problem. I forgot to remove